src/Frontend/Core/Header/MetaCollection.php

```diff
@@ -10,6 +10,17 @@
     /** @var MetaLink[] */
     private $metaLinks = [];
 
+    public function __construct()
+    {
+        /* @remark Sumocoders staging websites should not be tracked */
```

Comment:
Indeed this is probably the best place to set the meta tag. The __toString method gets called in the Twig template, so this way the change of accidentally overwriting the value is very low.